komodo/symlink/sanity_check.py

```diff
@@ -91,10 +91,21 @@ def assert_root_nodes(link_dict):
         )
 
 
-def _compare_dicts(dict_1, dict_2):
+def _sort_lists_in_nested_dicts(_dict: dict) -> dict:
+    result = {}
+    for key, value in _dict.items():
+        if isinstance(value, list):
+            result[key] = sorted(value)
+        else:
```

Comment:
Will there ever be a situation where there is another layer of dict? Should we check for this, and use this function recursively?

Reply:
I will remove `nested` from the function name. It was originally a recursive sorter, but that was before I understood the sorting thing was about lists..